packages/react/src/components/SuiteHeader/util/IdleTimer.js

```diff
@@ -77,6 +78,10 @@ class IdleTimer {
     )};expires=${expires};path=/;domain=${this.COOKIE_DOMAIN};`;
   }
 
+  deleteUserInactivityTimeout() {
+    document.cookie = `${this.COOKIE_NAME}=;Max-Age=0;path=/;domain=${this.COOKIE_DOMAIN};`;
```

Comment:
Do we need to set the "expires" to a date in the past?

Reply:
This is an example I found on stackoverflow to programmatically delete a cookie. If there is a better way, I don't mind changing it.

Reply:
no, I double checked and the `Max-Age=0` apparently does the same trick as setting the expires